qemu_v8.mk

```diff
@@ -161,9 +161,7 @@ TF_A_FLAGS ?= \
 	PLAT=qemu \
 	QEMU_USE_GIC_DRIVER=$(TFA_GIC_DRIVER) \
 	ENABLE_SVE_FOR_NS=2 \
-	ENABLE_SME_FOR_NS=2 \
 	ENABLE_SVE_FOR_SWD=1 \
-	ENABLE_SME_FOR_SWD=1 \
```

Comment:
Comment on "qemu_v8: enable SME for SPMC_AT_EL=2".
Why are these two lines not needed when SPMC_AT_EL=n? We still have QEMU_SME=on in that case.

Reply:
That's a good question. I'll restore the lines and remove the corresponding line below.